trio/_threads.py

```diff
@@ -81,7 +81,8 @@ def _do_it(self, cb, fn, *args):
             pass
         else:
             raise RuntimeError(
-                "this is a blocking function; call it from a thread"
+                "BlockingTrioPortal run() or run_async() was called from a "
```

Comment:
run_sync(), not run_async(). I don't think we want to call it "a trio.run() thread" because that confuses the terminology, so could I suggest instead something like the following? "must be called from a thread that is not inside a call to `trio.run`", or maybe "from a thread that is not running the Trio event loop".